### ISSUE TITLE
Segmentation loading with wadors not working

### ISSUE THREAD

**Issue by emelalkim** [dcmjs-org/dcmjs#358]
We are working on loading DICOM SEG objects with wadors loading using Cornerstone WadoImageLoader
WadoImageLoader version 4.13.0
It works with dcmjs version 0.12.4, but it cannot display with version 0.29.8
We narrowed it down to the changes [PR](https://github.com/dcmjs-org/dcmjs/pull/280)
When getting from the metadataProvider, instance retrieval returns undefined
```
          const sourceImageMetadata = metadataProvider.get(
              "instance",
              imageId
          );
```
which fails in Rows check
```
           if (
                Rows !== sourceImageMetadata.Rows ||
                Columns !== sourceImageMetadata.Columns
            ) {
```
Any help appreciated

**Comment by pieper** [comment]
I don't really know how that metadataProvider is supposed to work (maybe the API is documented somewhere about what you can expect it to provide and what not, but I didn't find it when I looked many years ago). But if I understand correctly from the commit and PR discussion, "instance" would refer to the full parsed part10 binary and that with rs that wouldn't exist like it does with uri.  Probably if the instance isn't available then and needed values should be parsed from the metadata.

**Comment by resulturan** [comment]
I think this issue can be closed because it was fixed with v0.30.1
